### PR TITLE
Bump gnosis-py to v3.8.0

### PR DESCRIPTION
**Pull request by Uxio0**
- Remove `pyethereum` dependency, as it's not needed anymore

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -19,9 +19,8 @@ djangorestframework==3.13.1
 djangorestframework-camel-case==1.3.0
 docutils==0.18.1
 drf-yasg[validation]==1.20.0
-ethereum==2.3.2
```
Comment (fmrsabino) [inline requirements.txt:22]: Why was this one removed?

Comment (Uxio0) [reply requirements.txt:22]: It was not being used, and also conflicting with the newest version of py-evm

Comment (fmrsabino) [reply requirements.txt:22]: For context I think it's better to leave a comment about these in the PR or in the commit message.

**Review by fmrsabino — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (Uxio0) [reply requirements.txt:22]: Makes sense, thanks

**Pull request merged**